lib/version-checker.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+const path = require('path');
```

Comment:
We should move this to where it is used.

Reply:
OK, I moved it all the way down into the `else if` block where it is needed.